src/lib/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection.Metadata;
+using System.Security.Cryptography;
```

Comment:
Do you still need this?

Reply:
I will double check and fix this and the other comment from Connie! Thanks a lot!!!